docs/integrating.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: integrating
+append_help_link: true
+description: >-
+  Learn about integrating Semgrep and the Semgrep Registry into your product.
+---
+
+# Semgrep integration guide for partners
+
+
+### Introduction
```

Comment:
```suggestion

```

There's no need to add "Introduction" to the top of a doc. It's an extra word users will skip over.